apps/backend/src/routes/signup/+page.svelte

```diff
@@ -0,0 +1,243 @@
+        </div>
+      </div>
+
+      <!-- Signup Form - Confirm Password Input -->
+      <div>
+        <label class="form-control w-full max-w-2xl">
+          <div class="label">
+            <span class="label-text">Confirm Password</span>
+            <span class="text-red-600 label-text-alt">*</span>
+          </div>
+          <input
+            type="password"
+            name="confirmpass"
+            placeholder="••••••••"
+            on:change={handleChange}
+            bind:value={$form.confirmPassword}
+            class="
+              input
+              input-bordered
+              w-full
+              max-w-2xl
+              focus:ring-secondary
+              focus:border-secondary
+              duration-300
+              focus:outline-none
+            " />
+        </label>
+        <div>
+          <span class="text-red-600">{$errors.confirmPassword}</span>
+        </div>
+      </div>
+
+      <div class="form-control mt-7">
+        <label class="label cursor-pointer">
+          <input
+            type="checkbox"
+            class="checkbox checkbox-secondary mr-5"
+            bind:checked={trigger}
+            on:click={toggleCheckbox} />
+          <span class="label-text"
+            >By signing up, you are creating a Armadillo account, and you agree to Armadillo's
+            <a class="text-secondary hover:text-accent duration-300" href="/login">
+              Terms of Use
+            </a>
+            and
+            <a class="text-secondary hover:text-accent duration-300" href="/login">
+              Privacy Policy
+            </a>
+          </span>
+        </label>
+      </div>
+
+      <!-- Signup Form - Submit/Continue Button -->
+      <button
+        type="submit"
+        class="
+					btn
+					btn-secondary
+					w-full
+					max-w-2xl
+					mt-8 mb-2
+					hover:ring-2
+					hover:ring-info
+				"
+        disabled={!trigger}
+        >Create Account
+      </button>
+    </form>
+
+    <!-- Signup Form - Link to Login -->
+    <p class="text-center">
+      Already have an account?
+      <a class="text-secondary hover:text-info duration-300" href="/login"> Log in </a>
+    </p>
+  </div>
+</div>
+
+<h1 class="text-4xl font-bold">Authentication Testing</h1>
+
+<div class="mt-4 grid grid-cols-2 gap-4">
+  <div class="flex flex-col gap-4">
+    <button class="btn btn-error w-full" on:click={() => signOut((error) => (apiError = error))}>
+      Sign Out
+    </button>
+    <h2 class="text-2xl">Form Data</h2>
+    <pre>Email Field   : {$form.email}</pre>
+    <pre>Password Field: {$form.password}</pre>
+
+    <h2 class="text-2xl">Auth Status</h2>
+    <p class="text-red-600">
+      If data below is <kbd class="kbd kbd-sm">null</kbd>, means there are no auth errors.
+    </p>
+    <pre>{JSON.stringify(apiError, null, 2)}</pre>
+
+    <h2 class="text-2xl">Auth User Data</h2>
+    <pre>{JSON.stringify($authStore, null, 2)}</pre>
+  </div>
+</div>
```

Comment:
Remove development code if you intend for this to be pushed into production

Reply:
Will not remove this yet as i still need the data to see if email is verified, this is for conditional logging in. To be done after this PR is resolved so just resolve this one first?